### ISSUE TITLE
[com.google.chromecast] The Cromecast won't stream a youtube video. The only thing that's appearing is a black screen with two white lines. After 5 seconds the Chromecast will return to the basic screensaver.

### ISSUE THREAD

**Issue by mroizo** [athombv/homey#2205]
**Metadata**

* **Homey Version**:
* **Homey Log Code**:
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**
* **Homey App Version**:
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**:

**What did you try to achieve?**

**What did you expect as result?**

**What was the result?**

**Any other remarks**

**Comment by Dijker** [comment]
Duplicate of #2196

**Comment by PetrichorXyz** [comment]
Any progress?

**Comment by PeterBox** [comment]
Have same problem !!Any progress?

**Comment by gijzo** [comment]
same problem here

**Comment by Pringman** [comment]
Same issue here. Tried it with two different Chromecast gen.1. Anyone knows if it helps buying a new chromecast or is that money down the bin?

**Comment by corvdniet** [comment]
+1

**Comment by gajdur** [comment]
@BasKiers @jeroenvollenbrock  can we help somehow to find the bug?

**Comment by Pringman** [comment]
Just tried it with with a newly bought CC gen.2, same issue. :(

**Comment by gajdur** [comment]
Maybe is this bug here: https://www.engadget.com/2018/01/19/google-rolls-out-patch-chromecast-wifi-bug/

**Comment by corvdniet** [comment]
I got the latest Chromecast firmware version 113131 via previewprogram  https://support.google.com/chromecast/answer/7124014?hl=nl. 
Does not solve the "two line"- issue :-(

**Comment by PetrichorXyz** [comment]
Just wondering if this problem is really a Chromecast issue. Apps like YouTube, NetFlix, etc. on phone casting to Chromecast are working fine. Seems to be a Homey issue.

**Comment by gooooba** [comment]
I agree. Everything else works with Chromecast except streaming from homey.

**Comment by DarkCinema** [comment]
New to this app and I am having the same issues. If any traces or debugs can help just let me know.
 I wanted to display a webpage via Homey using this plugin, but I just get the two white lines and the CAST_INIT_TIMEOUT.  

I am using a Gen2 Cromecast, connected on my AV Receiver. Casting the same webpage directly from a Windows 10 Client using Google Chrome works flawless.

**Comment by PetrichorXyz** [comment]
Is somebody still working on this? @BasKiers? Is there a need for logs or anything?

**Comment by corvdniet** [comment]
Bump
